src/plugins/needs-info/index.js

```diff
@@ -65,5 +53,5 @@ async function check(context) {
  */
 
 module.exports = robot => {
-    robot.on("issues.labeled", check);
+    robot.on("issues.closed", check);
```

Comment:
Should we search for a "Need Info" label in `payload.issue.labels` array instead of checking `payload.label` (which probably doesn't exist in this payload)?

`issues.labeled` was sending the added label as `payload.label`. but in this case the issue has been closed with an already existing label (which was added 7 days prior to closing the issue)?